### DISCUSSION TITLE
Footnotes in autogenerated PDF unnumbered (all 0)

### DISCUSSION THREAD

**Discussion by alyssaarbuckle** [knowledgefutures/pubpub#2731] [Support]
Hello, 

When I autogenerate a PDF version of an article from the journal I manage (IDEAH), the footnotes are not rendering correctly. Instead of being listed in a sequentially numbered list, all footnotes are marked with "0." Do you have any recommendations for how to fix this? 

Thanks!

**Comment by gabestein** [comment]
Hello — sorry for this issue, and for the delay getting back to you. I think this should now be resolved, but please feel free to re-open if it isn't (and if you don't mind, share the link where you're seeing the issue).

**Comment by alyssaarbuckle** [comment]
Hi Gabriel,

Thanks for looking into this! Unfortunately it is still an error on my end. For instance, if you navigate to the article https://ideah.pubpub.org/pub/qw0z19ss/release/1 and generate a PDF, the footnotes are all marked with "0." Please say if I've done something wrong on my end that I can amend! 

Best, 
Alyssa

**Reply by gabestein** [reply]
Ah, sorry — that's just because we needed to delete the old generated PDF from the cache. I've gone ahead and done that, and it looks like it's working now.

**Comment by alyssaarbuckle** [comment]
Brilliant! It's working on my end too :) Thank you for attending to this! 
-AA

**Comment by alyssaarbuckle** [comment]
Hello again Gabe, 

I'm wondering if you might need to delete further old generated PDFs from the cache to remedy this problem across the issue? Our special issue editor just pointed out to me that the error is persisting with the following articles in this issue: 
https://doi.org/10.21428/f1f23564.a384a7df
https://doi.org/10.21428/f1f23564.79d5a418
https://doi.org/10.21428/f1f23564.1572e06c
Let me know if there's anything I can do on my end to rectify this! 
-Alyssa

**Reply by gabestein** [reply]
I went ahead and cleared out cached PDFs across IDEAH to make sure there are no issues. The first time the downloads are requested may take a few seconds, but going forward you should be good to go.

**Reply by alyssaarbuckle** [reply]
Brilliant, thanks Gabe! All is well from my end now :) 
-AA
